interface/patient_file/download_template.php

```diff
@@ -166,6 +166,41 @@ function doSubs($s)
             $s = keyReplace($s, dataFixup($ptphone, xl('Phone')));
         } elseif (keySearch($s, '{PatientDOB}')) {
             $s = keyReplace($s, dataFixup(oeFormatShortDate($ptrow['DOB']), xl('Birth Date')));
+        } elseif (keySearch($s, '{PatientAge}')) {
+            // Calculate patient age
+            $dob = strtotime($ptrow['DOB']); // Convert date of birth to timestamp
```

Comment:
this will break for dob that are before 1970.
Rec using the getpatientAge() function in the patient.inc.php function library

Reply:
actually, should instead use the getPatientAge() function in the PatientService class